elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -434,7 +424,7 @@ export abstract class BaseTransaction {
 			this.getBasicBytes(),
 		]);
 
-		this._signatures.push(
+		this.signatures.push(
```

Comment:
Let's create another issue to update `sign` function

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/4906